lib/kitchen/directions/bake_stepwise.rb

```diff
@@ -4,12 +4,54 @@ module Kitchen
   module Directions
     module BakeStepwise
       def self.v1(book:)
+        # Block-level elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements
+        block_level = %w[
```

Comment:
might be nicer to define a method on ElementBase that does the "if element contains block-level element" logic